service/serializers.py

```diff
@@ -35,3 +35,54 @@ def to_representation(self, instance):
         ret['therapist'] = UserSerializer(instance.therapist).data
         ret['client'] = UserSerializer(instance.client).data
         return ret
+
+
+class ScheduleSerializer(serializers.ModelSerializer):
+    weekdays = {
```

Comment:
You already have this const. Why do you need to create it twice?

Reply:
Done